criu/files-reg.c

```diff
@@ -2133,6 +2133,11 @@ int open_path(struct file_desc *d, int (*open_cb)(int mntns_root, struct reg_fil
 
 	if (rfi->remap) {
 		if (!rfi->remap->is_dir) {
+			struct mount_info *mi = lookup_mnt_id(rfi->rfe->mnt_id);
+
+			if (mi && try_remount_writable(mi, true))
```

Comment:
pls check in the test that a test mount is read-only after restore.

Reply:
done:
zdtm: add ro-mount check after c/r to mntns_ghost01